DrewBradley/flashcards-starter
# Create, export, and test Card constructor in Card.js

#### What does this PR do?
Creates the Card class constructor and test it against card-test.js.

#### Where should the reviewer start?
Lines 1-10 of Card.js.

#### How should this PR be manually tested?
Run ```npm test test/card-test.js``` to check if all possible tests are passing.

#### Any additional considerations?
Not at the moment.
